OC4IDS_Indicator_Coverage.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<p>This needs changed to 'calculable'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Fixed in https://github.com/open-contracting/notebooks-oc4ids/pull/22/commits/bff12cda9068b62f2ac8b2203ece88ce0f005269</p>

Reply:
I think that commit fixes a different 'calcuable' typo - I think @odsrachel was flagging the one on line 390 (flagged inline below).